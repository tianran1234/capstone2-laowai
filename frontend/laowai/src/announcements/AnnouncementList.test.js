import React from "react";
import { render } from "@testing-library/react";
import Announcements from "./AnnouncementList";

it("matches snapshot", function () {
  const { asFragment } = render(<Announcements />);
  expect(asFragment()).toMatchSnapshot();
});
