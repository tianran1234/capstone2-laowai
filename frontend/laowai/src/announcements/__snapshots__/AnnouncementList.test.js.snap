// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h4>
    No announcement has been made.
  </h4>
</DocumentFragment>
`;
